### PR TITLE
feat(bridge): store intermediate state

### LINKED ISSUE #16: feat(bridge): store the intermediate state
Currently, the bridge application will lost its state (e.g. the latest block number) if it was killed by other reasons (e.g. process kill). It should store its intermediate state at separated storage (e.g. Redis, local SQLite3) atomically.

### PR DESCRIPTION
**Pull request by moreal**
It resolves #16.

### CODE REVIEW

**Comment by moreal** [comment]
Could you review this pull request? @longfin

**Comment by moreal** [comment]
PTAL @longfin  🙏🏻

File: bridge/src/sqlite3-monitor-state-store.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        )`;
+        return new Promise((resolve, error) => {
+            database.run(CREATE_TABLE_QUERY, e => {
+                if (e) {
+                    error();
+                } else {
+                    resolve();
+                }
+            });
+        });
```
Comment (longfin) [inline bridge/src/sqlite3-monitor-state-store.ts:34]: How about using [util.promisify()](https://nodejs.org/dist/latest-v8.x/docs/api/util.html)?

**Review by longfin — APPROVED** [review]

File: bridge/src/sqlite3-monitor-state-store.ts (hunk 1 above)
Comment (moreal) [reply bridge/src/sqlite3-monitor-state-store.ts:35]: I applied it in e9bcab4.

**Pull request merged**